code/api/src/modules/order/model.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Order
+module.exports = function(sequelize, DataTypes) {
+  let Order = sequelize.define('orders', {
+    subscriptionId: {
+      type: DataTypes.INTEGER
+    },
+    deliveryDate: {
+      type: DataTypes.DATE
+    },
+    status: {
+      type: DataTypes.STRING
+    }
+  })
+
+  Order.associate = function(models) {
+    Order.belongsTo(models.Subscription)
+    Order.hasMany(models.OrderProduct)
+    // Order.belongsToMany(models.User, {through: models.Subscription})
+    // Order.belongsToMany(models.Product, {through: models.OrderProduct})
```

Comment:
We could probably remove this commented out code now that we figured out the nested issue we were running into earlier. :)